lib/pharos/phases/configure_kube_proxy.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureKubeProxy < Pharos::Phase
```

Comment:
Not needed anymore.. probably leftover from rebase?

Reply:
Apparently. Removed.